onnxruntime/core/optimizer/transpose_optimization/onnx_transpose_optimization.cc

```diff
@@ -1172,29 +1172,36 @@ static bool HandleUnsqueeze(HandlerArgs& args) {
 
 constexpr HandlerInfo unsqueeze_handler = {&FirstInput, &HandleUnsqueeze};
 
-static bool HandleQuantizeDequantizeScale(const api::GraphRef& graph, const std::vector<int64_t>& perm,
-                                          api::NodeRef& node, int64_t opset) {
-  if (opset >= 13) {
-    size_t rank = perm.size();
-    // Update axis in Opset >= 13 if scale/zero_point are non-scalar
-    auto inputs = node.Inputs();
+bool TransposeQuantizeDequantizeAxis(const api::GraphRef& graph, const std::vector<int64_t>& perm, api::NodeRef& node) {
+  size_t rank = perm.size();
 
-    auto inp_shape = graph.GetValueInfo(inputs[1])->Shape();
-    bool scalar_params = inp_shape.has_value() && inp_shape->size() == 0;
+  // Update axis if scale/zero_point are non-scalar
+  auto inputs = node.Inputs();
 
-    if (!scalar_params) {
-      int64_t axis = node.GetAttributeIntDefault("axis", 1);
-      if (!NormalizeAndValidateAxis(axis, rank)) {
-        return false;
-      }
-      node.SetAttributeInt("axis", perm[gsl::narrow_cast<size_t>(axis)]);
+  auto inp_shape = graph.GetValueInfo(inputs[1])->Shape();
+  bool scalar_params = inp_shape.has_value() && inp_shape->size() == 0;
+
+  if (!scalar_params) {
+    int64_t axis = node.GetAttributeIntDefault("axis", 1);
+    if (!NormalizeAndValidateAxis(axis, rank)) {
+      return false;
     }
+    node.SetAttributeInt("axis", perm[gsl::narrow_cast<size_t>(axis)]);
+  }
+  return true;
+}
+
+constexpr bool HandleQuantizeDequantizeAxis(const api::GraphRef& graph, const std::vector<int64_t>& perm,
+                                            api::NodeRef& node, int64_t opset) {
+  if (opset >= 13) {
```

Comment:
```suggestion
  if (opset < 13) {
    // no `axis` value until opset 13
    return true;
  }
  
  return TransposeQuantizeDequantizeAxis(graph, perm, node);;
```

nit: a comment saves a new developer having to go read the ONNX spec changes to figure out why we're returning true by default when typically we return 'false' if something isn't handled. Flipping the `if` condition (at least to me) makes it read a bit more naturally as well.